### PR TITLE
Add P2P, Network, and Qt categories to the desktop icon

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: contrib/debian/bitcoin-qt.desktop [hunk 1]
```diff
@@ -10,5 +10,5 @@ Terminal=false
 Type=Application
 Icon=bitcoin128
 MimeType=x-scheme-handler/bitcoin;
-Categories=Office;Finance;
+Categories=Office;Finance;P2P;Network;Qt;
```
Comment (laanwj) [inline contrib/debian/bitcoin-qt.desktop]: I don 't think `Qt` is a useful category. It's like adding `C++`.

Comment (luke-jr) [reply contrib/debian/bitcoin-qt.desktop]: The `Qt` category is defined as "Application based on Qt libraries", which certainly applies to bitcoin-qt.

https://specifications.freedesktop.org/menu-spec/latest/apas02.html

Comment (laanwj) [reply contrib/debian/bitcoin-qt.desktop]: Huh, okay.
Well if it's part of the standard then ok...

**Comment by laanwj** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/12854/commits/b63f23c175785450b4665f9fba75cdf64171e236

**Comment by Sjors** [comment]
Is Debian the only distro supporting this standard?

**Comment by laanwj** [comment]
freedesktop standards in principle apply to all distributions and window managers.
So it would make sense to move it from `contrib/debian` to a place where `make install` can install it (not in thie PR though).

**Pull request merged**